File: 1553-count-triplets-that-can-form-two-arrays-of-equal-xor/solution.java
class Solution {
    public int countTriplets(int[] arr) {
        int c=0,n=arr.length;
        for(int i=0;i<n;i++)
        {
            for(int j=i+1;j<n;j++)
            {
                int a=0;
                for(int k=i;k<j;k++)
                {
                    a=a^arr[k];
                }
                int b=0;
                for(int k=j;k<n;k++)
                {
                    b=b^arr[k];
                    if(a==b)
                    {
                        c++;
                    }
                }
                
            }
        }
        return c;
    }
}

File: 1553-count-triplets-that-can-form-two-arrays-of-equal-xor/solution.c
int countTriplets(int* arr, int arrSize) {
    int c=0;
    for(int i=0;i<arrSize;i++)
    {
        for(int j=i+1;j<arrSize;j++)
        {
            int a=0;
            for(int k=i;k<j;k++)
            {
                a^=arr[k];
            }
            int b=0;
            for(int k=j;k<arrSize;k++)
            {
                b^=arr[k];
                if(a==b)
                {
                    c++;
                }
            }
            
        }
    }
    return c;
}
